lib/godmin/helpers/forms.rb

```diff
@@ -4,6 +4,11 @@ module Forms
       def form_for(record, options = {}, &block)
         super(record, { builder: FormBuilders::FormBuilder, inline_errors: false }.merge(options), &block)
       end
+
+      def select_tag(name, option_tags = nil, options = {})
```

Comment:
Not sure we should override stuff outside of the form builder. If you want the bootstrap stuff and godmin stuff outside a `form_for` you could always specify the form builder for your form manually.

Thoughts?